src/components/timeout-dialog/dialog.test.js

```diff
@@ -1,5 +1,4 @@
 /* eslint-env jest */
-import mock from 'jest-mock';
```

Comment:
jest is available globally and has the mock functions in it, only need jest-mock if you're not using jest (I think)